src/systems/diffeqs/sensitivity_transform.jl

```diff
@@ -0,0 +1,20 @@
+function forward_sensitivity_transform(sys::ODESystem,p::AbstractVector{<:Expression})::ODESystem
+    iv  = sys.iv()
+    f = du_dt = [eq.rhs for eq ∈ sys.eqs]
+    u = [state(iv) for state ∈ states(sys)] 
+    sen_symbols = [Symbol(:d,state.op.name,:_d,par.op.name) for state ∈ u, par in p]
+    du_dp = [Variable(sen_symbol)(iv) for sen_symbol ∈ sen_symbols]
+    @derivatives D'~iv
```

Comment:
```suggestion
    D = Differential(iv.name)
```

so the naming is kept.